src/skin.cpp

```diff
@@ -245,16 +245,11 @@ void SkinInfillAreaComputation::applySkinExpansion(const Polygons& original_outl
  *
  * this function may only read/write the skin and infill from the *current* layer.
  */
-void SkinInfillAreaComputation::generateSkinInsets(SliceLayerPart* part)
+void SkinInfillAreaComputation::generateSkinInsetsAndInnerSkinInfill(SliceLayerPart* part)
```

Comment:
Do you think I should change the wording?
I can't really come up with something better in the couple of seconds I tried... :P